site/templates/page/page-header.jet

```diff
@@ -5,9 +5,9 @@
         <s72-plan-label data-slug="{{plan.Slug}}" data-title="{{plan.Name}}"></s72-plan-label>
       {{end}}
       <h1>{{title}}</h1>
-      {{if plan != 0 && plan.HasExpiryDate() }}
-        {{ if plan.HasExpiryDate() }}
-          <p class="alert p-0 border-0 my-1">
```

Comment:
My opinionated take is this is a step in the wrong direction

Reply:
How so?

Reply:
Ping @matthewthomsonnz - what's the step in the wrong direction?

Reply:
```
.curated-plan-page {
  .page-header {
    .alert {
      @extend .p-0;
      @extend .border-0;
```
I know inline utility classes are controversial, but I think they are a good solution for one offs, and mitigate nested css hell

Reply:
Ah, the removal of the utility classes are the step in the wrong direction. I'm not against utility classes per-se and `@extend` can lead to more bloat in the final CSS. In this case I agree with you, but it's pretty marginal so I don't mind either way.

Reply:
PS, reason why it was hard to spot the problem, is the view on the "Files changed" tab points out the utility classes while the one in "Conversation" doesn't:

![image](https://user-images.githubusercontent.com/2253814/157368651-c8629914-6000-48ba-8397-7286bd62e7a0.png)

![image](https://user-images.githubusercontent.com/2253814/157368735-2e55f8ed-0c9b-475d-8389-73789e74e271.png)

Reply:
@matthewthomsonnz Interested in your feedback now